public/locales/de/translations.json

```diff
@@ -0,0 +1,49 @@
+	"roadmap.organization.2": "Flakes will be standardized with its current implementation as a v0. While not ideal, the feature is used far too widely to be changed or removed without breaking the ecosystem. Instead, this v0 implementation will be enabled and future work for Flakes that addresses its shortcomings may be handled by a Flakes SIG.",
+	"roadmap.organization.3": "The `aux` CLI will be modified to provide more ergonomic management of packages and systems. Additional subcommands such as `aux system switch` and `aux system build` will be added to make onboarding and ongoing maintenance easier.",
+	"roadmap.infra.title": "Infrastruktur",
+	"roadmap.infra.1": "Now that the project has significantly diverged from upstream, we will need to provide our own Continuous Integration and Binary Cache services. Existing governance structures will be used to manage the adoption of these technologies.",
+	"roadmap.infra.2": "Documentation will be included as a part of the Continuous Integration process and will have Continuous Deployment configured to keep it up to date, making it easier for contributors to create educational materials.",
+	"roadmap.infra.3": "Search infrastructure for packages and options will also be created.",
+	"roadmap.alignment.title": "Un­ab­hän­gig­keit",
+	"roadmap.alignment.1": "By this point, we've done it. We have created a sustainable, independent fork of the Nix ecosystem. Now that the initial work has been completed we will begin the governance rotation process with our first elections and education as to how project governance works.",
+	"roadmap.alignment.2": "Documentation, project branding, and packages will be polished and presented as a fully stable release. From this point forward the Steering Committee will be managing the creation of Special Interest Groups and other necessary Committees. All enhancements will be suggested in Aux Enhancement Proposals which encourage existing SIGs to implement features or encourage the formation of a new SIG to be created for such a purpose.",
+
+	"values.title": "Die Werte",
+	"values.rep.title": "Vertretung",
+	"values.rep": "Die Ansichten der Mitwirkenden sollten vertreten werden und die Organisationsstruktur sollte demokratisch ausgerichtet sein.",
+	"values.kindness.title": "Wohlwollen",
+	"values.kindness": "Das Projekt sollte einladend sein und einen sicherer Ort für die Teilnahme darstellen.",
+	"values.collab.title": "Zusammenarbeit",
+	"values.collab": "Kommunikation und Zusammenarbeit sollten gefördert werden.",
+	"values.sustainability.title": "Nachhaltigkeit",
+	"values.sustainability": "Arbeitsbelastung, Finanzen, Umfang und Burnout sollten sorgfältig abgewogen werden, um sicherzustellen, dass das Projekt stabil ist.",
+	"values.accessibility.title": "Offenheit",
+	"values.accessibility": "Elitäres Verhalten ist unerwünscht, unser Wissen soll geteilt werden und wir sollten die Werkzeuge bereitstellen, die jedem zum Erfolg und Lernen verhelfen."
```

Comment:
In a similar spirit to 'soll->sollte': ... und wir *wollen* die Werkzeuge ...

Reply:
Isn't 'sollte' a weaker form of 'soll' in this context? I found 'soll' better matching as the original states "our knowledge is here to be shared", which I read as "sharing is mandatory".

+1 on 'wollen'

Reply:
> Isn't 'sollte' a weaker form of 'soll' in this context? I found 'soll' better matching as the original states "our knowledge is here to be shared", which I read as "sharing is mandatory".

Oh, I actually meant only the change to 'wollen'. With 'soll->sollte' I was just referring to a similar change above (which exchanged 'sollte' with 'soll'). But even now it took me several attempts at finding the mistake in what I wrote: I mean 'sollte->soll', but accidentally wrote it the other way round. Sorry for the confusion.

Reply:
Thanks for the clarification, double-checked&fixed other occurrences as well.